addons/hr_expense_project/models/project.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.addons.project.models.project import _to_action_data
+
+class Project(models.Model):
+    _inherit = 'project.project'
+
+    expenses_count = fields.Integer('# Expenses', compute='_compute_expenses_count', default=0, groups='hr_expense.group_hr_expense_team_approver')
+    show_expenses_count_button = fields.Boolean('Show the expenses stat button on the project view form and project update right panel?', compute='_compute_show_expenses_count_button', default=False)
```

Comment:
With this task, stat buttons were added on the side panel of the project updates and I did not find any convenient way to do this without duplicating all the stat buttons. After discussion with the team I decided to create this field in order to reuse it in both places and avoid duplicating the group/count (in order to avoid maintainability issues in the future if we decide to change the group or the visibility conditions).

Would you rather duplicate this information anyway and get rid of the computed field?

Reply:
I find it strange to alter the models with several fields only to avoid displaying 2 buttons each time.

Before refactoring the code, I would like to have the opinion of @IT-Ideas or @xavierbol